Readme.md

```diff
@@ -88,6 +88,15 @@ where `[liquid_domain]` is the value of `liquid.domain` from the `liquid.ini` fi
 
 
 ## Configuration
+### Use Pipenv
+```shell
+sudo apt update
+sudo apt-get install python3-venv python3-pip git curl unzip
+pip3 install pipenv
+pipenv install -r requirements.txt
```

Comment:
The typical way to use `pipenv` is with a `Pipfile` instead of a `requirements.txt` file. Please read their documentation.